mxcubeweb/core/components/user/usermanager.py

```diff
@@ -95,8 +84,11 @@ def update_active_users(self):
                     f"Logged out inactive user {_u.username}"
                 )
                 self.app.server.user_datastore.deactivate_user(_u)
+                self.app.server.emit(
+                    "userChanged", room=_u.socketio_session_id, namespace="/hwr"
+                )
```

Comment:
Send `userChanged` only to users who have been "deactivated".

Reply:
Oh FYI, I tried to send `observersChanged` here only when at least one user had been deactivated, but then existing users weren't getting notified when a new observer logged in. This is not great, but didn't want to risk breaking things, so I didn't go any further.

Reply:
I see, that is a bit odd